python/ray/_private/state.py

```diff
@@ -744,7 +744,7 @@ def _live_node_ids(self):
         """Returns a set of node IDs corresponding to nodes still alive."""
         return {node["NodeID"] for node in self.node_table() if (node["Alive"])}
 
-    def _available_resources_per_node(self):
```

Comment:
Should we just keep this as it is? There's no reason to keep backward compatibility here, but there's also nothing to lose

Reply:
It's  no longer a private function so we should remove the `_`.